### ISSUE TITLE
[search] Binary content indexing is skipped when file size is too large

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#5997]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

The following method skips binary file indexing when content is too large
https://github.com/craftercms/search/blob/5efc63c94f63a87a021d929613718624b7bb22b3/crafter-search-batch-indexer/src/main/java/org/craftercms/search/batch/impl/AbstractBinaryFileBatchIndexer.java
and
https://github.com/craftercms/search/blob/b8e73f5a2781a7479b65bcdaf2acbcb91262f31e/crafter-search-batch-indexer/src/main/java/org/craftercms/search/batch/impl/AbstractBinaryFileWithMetadataBatchIndexer.java

Files should not be skipped but instead, their binary content should be excluded.

### Steps to reproduce

Steps:

1. Upload a binary asset with size greater than the configured max size
2. Note the skip message or attempt to find the asset in the authoring or delivery index

### Relevant log output

```shell
2023-03-19 20:44:22.857  WARN 42 --- [deployment-19] stractBinaryFileWithMetadataBatchIndexer : Skipping large binary file @ /static-assets/content/videos/big-video.mp4
 2023-03-19 20:44:22.857  INFO 42 --- [deployment-19] l.processors.AbstractDeploymentProcessor : ----- </ authoringElasticsearchIndexingProcessor @ mysite-authoring > -----
 2023-03-19 20:44:22.857  INFO 42 --- [deployment-19] l.processors.AbstractDeploymentProcessor : ----- < fileOutputProcessor @ mysite-authoring > -----
```

**Comment by russdanner** [comment]
A possible workaround is to increase the max size allowed.

**Comment by yacdaniel** [comment]
Verified fixed in 4.1-SNAPSHOT.

Verified binary indexed correctly, search by filename, metadata shows in filters. 

![image](https://github.com/craftercms/craftercms/assets/6722074/ade1c9ff-4dc3-4cc9-913f-03852de973a5)
